includes/wp-admin/class-welcome-fields.php

```diff
@@ -194,22 +196,51 @@ public static function render_author_profile_section() {
 	 * Render troubleshooting section.
 	 */
 	public static function render_troubleshooting_section() {
+		$results = Health_Check::count_results();
 		?>
-		<p>
-			<?php
-			echo wp_kses(
-				\sprintf(
-					/* translators: the placeholder is the Site Health URL */
-					\__(
-						'If you have problems using this plugin, please check the <a href="%s">Site Health</a> page to ensure that your site is compatible and/or use the "Help" tab (in the top right of the settings pages).',
-						'activitypub'
+		<div class="notice notice-warning inline">
+			<p>
+				<span class="dashicons dashicons-warning"></span>
+				<?php
+				echo wp_kses(
+					\sprintf(
+						/* translators: the placeholders are the number of critical and recommended issues on the site. */
+						\__(
+							'<strong>Important:</strong> You have <span class="count">%1$d</span> critical and <span class="count">%2$d</span> recommended issues on your site. These issues may affect your site\'s compatibility with the fediverse.',
```

Comment:
```suggestion
							'<strong>Important:</strong> There are <span class="count">%1$d</span> critical and <span class="count">%2$d</span> recommended issues affecting your site&#8217;s compatibility with the fediverse.',
```
Let's make it less personal